### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-it/conversationlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#192]
```
translated line with "UNTRANSLATED" (from your reply to issue 185)
and added about 50 missing rows
```

Original issue reported on code.google.com by `liogiu@gmail.com` on 7 May 2011 at 6:56

Attachments:
- [conversationlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-192/comment-0/conversationlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thank you again liogiu! You work fast and produce high-quality :)
Committed in r80.
```

Original comment by `oskar.wi...@gmail.com` on 7 May 2011 at 7:27
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.9**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:52
- Changed state: **Done**
